documentation/markdown_files/installing.markdown

```diff
@@ -63,6 +63,15 @@ You are now in a virtual environment, to exit it you may call `deactivate`, to e
 
 ## Hotdoc itself
 
+### Python dependencies
+
+It is recommended to use the locked versions of all dependencies which hotdoc has been tested against:
```

Comment:
I think I'd word this slightly more neutrally, eg:

`In order to use the exact versions of all dependencies hotdoc is known to work against:`

Reply:
Tweaked.